documents/src/pages/elements/counter.md

```diff
@@ -145,3 +145,10 @@ Compact notation also work with the `max` attribute.
 <ef-counter max="90000000000000" value="99999999999999"></ef-counter>
 <ef-counter max="900000000000000" value="999999999999999"></ef-counter>
 ```
+
+## Accessibility
+::a11y-intro::
+
+`ef-counter` can be read by screen reader in a similar way as a generic text HTML element. It doesn't meant to be an actionable element so, generally, it shouldn't be focusable and not necessary to be assigned with any roles.
```

Comment:
This is correct and that means `counter` shouldn't be in **Inputs** category. IMO it should be in Data Container

Reply:
Tend to agree with you Phet but we need to sync with UX team so they can update their design system doc to be in sync with us.